code/game/turfs/simulated/floor/light_floor.dm

```diff
@@ -77,14 +77,13 @@
 
 /turf/simulated/floor/light/attackby(obj/item/C, mob/user, params)
 	if(istype(C,/obj/item/light/bulb)) //only for light tiles
-		if(istype(builtin_tile, /obj/item/stack/tile/light))
-			if(!state)
-				qdel(C)
-				state = LIGHTFLOOR_ON
-				update_icon()
-				to_chat(user, "<span class='notice'>You replace the light bulb.</span>")
-			else
-				to_chat(user, "<span class='notice'>The light bulb seems fine, no need to replace it.</span>")
+		if(!state)
+			qdel(C)
+			state = LIGHTFLOOR_ON
+			update_icon()
+			to_chat(user, "<span class='notice'>You replace the light bulb.</span>")
+		else
+			to_chat(user, "<span class='notice'>The light bulb seems fine, no need to replace it.</span>")
```

Comment:
Not really related to this PR, but do we even need this message? Not a whole load of actions will say "no need to X" when you do them (Only one I can think of is 'the airlock doesnt need repairing')